src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -619,6 +638,18 @@ uint32_t Subgraph::getBroadcastingMask(const std::vector<VectorDims>& input_shap
     return mask;
 }
 
+std::set<size_t> Subgraph::getConstantInputIndexes() const {
+    // Add constant path support
```

Comment:
I don't understand this comment. Could you please remove it or provide more details?

Reply:
Added the ticket and more details in the comment. Thanks!